README.md

```diff
@@ -30,259 +30,25 @@ After you're finished please remove all the comments and instructions!
   <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
   <br/>
 
-  <h3><b>Microverse README Template</b></h3>
+  <h3><b>Hello World Microverse</b></h3>
 
```

Comment:
- [ ] Kindly include a very good description in your pull request, describing what you've done in this milestone.

![descriptive](https://user-images.githubusercontent.com/41163571/234657304-4f49f492-2d22-4edc-9946-955b56671ed5.gif)